tests/python_client/milvus_client/test_milvus_client_timestamptz.py

```
@@ -1308,4 +1099,297 @@ def test_milvus_client_timestamptz_add_field_not_nullable(self):
+        """
+        target:  Milvus raise error when add field with default value for timestamptz field
+        method:
+            1. Create a collection with default value for timestamptz field
+            2. Add field with invalid default value for timestamptz field
+        expected: Step 1 should result fail
+        """
+        # step 1: create collection
+        client = self._client()
+        collection_name = cf.gen_collection_name_by_testcase_name()
+        schema = self.create_schema(client, enable_dynamic_field=False)[0]
+        schema.add_field(default_primary_key_field_name, DataType.INT64, is_primary=True, auto_id=False)
```

Ruff (0.14.10)

`DataType` may be undefined, or defined from star imports

(F405)